jenkins_pipelines/environments/common/pipeline.groovy

```diff
@@ -148,16 +148,25 @@ def run(params) {
                     try {
                         sh "./terracumber-cli ${common_params} --logfile ${resultdirbuild}/testsuite.log --runstep cucumber --cucumber-cmd 'cd /root/spacewalk/testsuite; ${env.exports} rake cucumber:finishing'"
                     } catch(err) {
-                        println("ERROR: rake cucumber:finishing failed")
+                        println("ERROR: rake cucumber:finishing failed: ${err}")
                         error = 1
                     }
                     try {
                         sh "./terracumber-cli ${common_params} --logfile ${resultdirbuild}/testsuite.log --runstep cucumber --cucumber-cmd 'cd /root/spacewalk/testsuite; ${env.exports} rake utils:generate_test_report'"
                     } catch(err) {
-                        println("ERROR: rake utils:generate_test_repor failed")
+                        println("ERROR: rake utils:generate_test_repor failed: ${err}")
                         error = 1
                     }
                     sh "./terracumber-cli ${common_params} --logfile ${resultdirbuild}/testsuite.log --runstep getresults"
+                    // In the case of an AWS environment, we want to export the reports to a publicly accessible web server.
+                    if (params.sumaform_backend == "aws") {
+                        try {
+                            sh "./terracumber-cli ${common_params} --logfile ${resultdirbuild}/webserver.log --runstep cucumber --cucumber-cmd 'rsync -av ${resultdirbuild} /mnt/www/'"
```

Comment:
THIS IS WRONG.
This path `resultdirbuild` it's only available on the Jenkins workspace, not on the controller. I'm thinking how to fix it...